### DISCUSSION TITLE
Captions on Images for LaTeX

### DISCUSSION THREAD

**Discussion by iwelch** [jgm/pandoc#10217] [General]
```
![testfig]
[testfig]: test.pdf "hello, why am I not the caption?"
```

The long string does not do much in latex.  The caption is testfig.  testfig also serves as the reference, identifying the correspondence, like a label.  this means changing the caption means it has to be changed twice, which can be error prone.

Somehow, this seems "off."  It would be nice to be able to specify a caption, say

```
[testfig]: test.pdf "long ignored string" caption="hello, why am I not the caption?"
```

**Comment by jgm** [comment]
If you search issues, you'll find a couple of issues about this general topic.
